### PR TITLE
Initial checkin.  Scripts to prep & install docker for Docker-Swarm + v2plugin.

### PR DESCRIPTION
**Pull request by dojacobs**
These scripts will be run by Jenkins when setting up Docker-Swarm using v2plugin (instead of the older "legacy" method.)

### CODE REVIEW

File: cluster/docker17/centos_docker_install.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# the other one. Fill in the version you want.
+yum -y install docker-ce
+#sudo yum install docker-ce-<VERSION>
+
+# Post-install steps
+# add admin user to docker group 
+usermod -aG docker $SUDO_USER
+
+# add /etc/docker/ if it doesn't exist
+if [ ! -d /etc/docker ]; then
```
Comment (dseevr) [inline cluster/docker17/centos_docker_install.sh]: This whole `if` statement can be replaced by `mkdir -p /etc/docker`.  It returns 0 even if the directory already exists

File: cluster/docker17/centos_docker_install.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    mkdir /etc/docker
+fi
+
+# add (and create) daemon.json with entry for storage-device
+    cat <<EOT >> /etc/docker/daemon.json
```
Comment (dseevr) [inline cluster/docker17/centos_docker_install.sh]: Unnecessary indentation at the beginning of the line

File: cluster/docker17/centos_docker_install.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+EOT
+
+# start up docker
+systemctl enable docker
+systemctl start docker
+
+# TODO: REMOVE THIS ONCE THE Contiv Installer DOES THIS
+# make sure /etc/openvswitch exists
+if [ ! -d /etc/openvswitch ]; then
```
Comment (dseevr) [inline cluster/docker17/centos_docker_install.sh]: `mkdir -p /etc/openvswitch`

File: cluster/docker17/centos_docker_install.sh [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    mkdir /etc/openvswitch
+fi
```
Comment (dseevr) [inline cluster/docker17/centos_docker_install.sh]: `exit 0` at end of script to make it the same as the other one

**Comment by dojacobs** [comment]
Updated.

**Pull request merged**